### DISCUSSION TITLE
How to put multiple embedded REPL in iframes?

### DISCUSSION THREAD

**Discussion by mafshin** [jupyterlite/jupyterlite#1276] [Q&A]
Hi All,

I followed the [tutorial](https://jupyterlite.readthedocs.io/en/latest/quickstart/embed-repl.html) to embed the REPL in a website. The following error appeared in console, any idea about the root cause?

Also while trying to put multiple iframes with REPL in the same page, only one of them executes the code and works properly, is there any configuration to make it work?

```
index.es6.js:2014 Plugin '@jupyter-notebook/lab-extension:interface-switcher' failed to activate.
(anonymous) @ index.es6.js:2014
index.es6.js:2015 TypeError: No provider for: @jupyterlab/notebook:INotebookTracker.
    at u.resolveRequiredService (index.es6.js:1932:19)
    at index.es6.js:1845:56
    at Array.map (<anonymous>)
    at u.activatePlugin (index.es6.js:1845:42)
    at index.es6.js:2013:25
    at Array.map (<anonymous>)
    at u.start (index.es6.js:2012:35)
    at P (index.js:442:13)
```

**Comment by jtpio** [comment]
> The following error appeared in console, any idea about the root cause?

This error should be harmless and comes from the `notebook` package. But could be fixed to avoid noise in the dev tools console.

> Also while trying to put multiple iframes with REPL in the same page, only one of them executes the code and works properly, is there any configuration to make it work?

Hmm that's odd. Normally the IFrames should be isolated. Is there any error in the dev tools console?
